test/parallel/test-repl-persistent-history.js

```diff
@@ -247,10 +247,7 @@ function runTest(assertCleaned) {
     useColors: false,
     terminal: true
   }, function(err, repl) {
-    if (err) {
-      console.error(`Failed test # ${numtests - tests.length}`);
```

Comment:
Adding a comment indicating that the console.log should not be removed would be helpfuil